tuxemon/core/components/menu/save_menu.py

```diff
@@ -40,12 +40,12 @@ def draw(self):
             slot_name = "slot" + str(slot_num)
 
             # Check to see if a save exists for the current slot
-            if os.path.exists("saves/slot" + str(slot_num) + ".png"):
+            if os.path.exists(".saves/slot" + str(slot_num) + ".png"):
```

Comment:
For what it is worth, I have all the known input bugs sorted out on my
branch. It may not be worth the effort to fix them here.

On Tue, Jan 5, 2016 at 12:57 AM superman2k5 <notifications@github.com>
wrote:

> In tuxemon/core/components/menu/save_menu.py
> <https://github.com/Tuxemon/Tuxemon/pull/89#discussion_r48816959>:
>
> > @@ -40,12 +40,12 @@ def draw(self):
> >              slot_name = "slot" + str(slot_num)
> >
> >              # Check to see if a save exists for the current slot
> > -            if os.path.exists("saves/slot" + str(slot_num) + ".png"):
> > +            if os.path.exists(".saves/slot" + str(slot_num) + ".png"):
>
> Makes sense, this was part of my questions I had when I was asking about
> the folder in the first place.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Tuxemon/Tuxemon/pull/89/files#r48816959>.
>